vehicle/SUV.h

```diff
@@ -1,12 +1,13 @@
 #ifndef SUV_H
 #define SUV_H
 
+#include <memory>
```

Comment:
not needed here, alredy included in Car.h

Reply:
removed